packages/node_modules/@webex/plugin-meetings/src/meetings/util.js

```diff
@@ -64,6 +64,23 @@ MeetingsUtil.checkForCorrelationId = (deviceUrl, locus) => {
   return false;
 };
 
+MeetingsUtil.parsePreferedWebexSite = (userPreferences) => {
```

Comment:
`parsePreferedWebexSite` can be changed to `parsePreferredWebexSite`

Reply:
done